src/components/QrScan/QrScan.js

```diff
@@ -1,32 +1,74 @@
 import React, { useState } from 'react';
-import { Button, Grid } from '@mui/material';
+import { Button, Grid, Box } from '@mui/material';
 import { makeStyles } from '@mui/styles';
 import QrReader from 'react-qr-reader';
 import { useHistory } from 'react-router-dom';
 import { v4 as uuidv4 } from 'uuid';
 import frame from 'assets/frame.png';
 import { useQrDataContext } from 'components/QrDataProvider';
 
-const useStyles = makeStyles(() => ({
+const useStyles = makeStyles((theme) => ({
   button: {
     '&:hover': {
       cursor: 'default'
     }
   },
+  box: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    width: '100%',
+    marginTop: '2em',
+  },
+  grid: {
+    display: 'flex',
+    flexDirection: 'column',
+    flexWrap: 'nowrap',
+    alignItems: 'center',
+    justifyContent: 'center',
+    width: 'auto',
+    [theme.breakpoints.down('md')]: {
+      maxHeight: '550px',
+      maxWidth: '300px',
+      margin: '1rem',
+    },
+    [theme.breakpoints.up('md')]: {
+      maxHeight: '550px',
+      maxWidth: '650px',
+      margin: '2rem',
+    }
+  },
+  gridContainerMultiple: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'right',
+    justifyContent: 'right',
+    paddingBottom: '1rem',
+  },
+  gridItem: {
+    display: 'flex',
+    position: 'relative',
+    alignItems: 'center',
+    justifyContent: 'center',
+    width: '100%',
+    height: '100%',
+  },
   frame: {
     position: 'relative',
-    height: '550px',
-    width: '640px',
-    marginTop: '3rem',
-    marginBottom: '3rem',
+    [theme.breakpoints.down('md')]: {
+      maxHeight: '550px',
+      maxWidth: '300px',
+    },
+    [theme.breakpoints.up('md')]: {
+      maxHeight: '550px',
+      maxWidth: '650px',
+    },
+    objectFit: 'contain',
     zIndex: '2',
   },
   qrScanner: {
     position: 'absolute',
-    marginLeft: '2.2rem',
-    marginTop: '4.5rem',
-    height: '500px',
-    width: '570px',
+    width: '80%',
```

Comment:
It looks like the camera no longer fits well in the outlined box. Setting the width and the height to 90% makes it fit as it did previously. I do realize that this might change when resolving merge conflicts, due to the update in the QR Scanner which is now in the main branch.

```suggestion
    width: '90%',
    height: '90%',
```

Reply:
I stretched the width to 90% but left the height out. Stretching height to 90% changes the aspect ratio and distorts the QR so it looks more like a rectangle than a square.